message_ix/util/__init__.py

```diff
@@ -14,28 +15,31 @@
 
 
 def copy_model(
-    path: Path, overwrite: bool = False, set_default: bool = False, quiet: bool = False
+    path: Path,
+    source_dir: Optional[Path] = None,
```

Comment:
If our own downstream code, or user code, has for instance:
```py
copy_model("/foo/bar", False, False, True)
```
…then inserting a new positional-or-keyword argument before other positional-or-keyword arguments will break that code.

In point of fact, such code likely doesn't exist. But in general, good practice is to follow a sequence like:
1. Add new arguments at the end of the list. Possibly add them as keyword-only:
   ```py
   ...
   quiet: bool = False,
   *,
   source_dir: Optional[Path] = None,
   ...
   ```
2. Possibly, later make certain other arguments keyword-only. (This is a pattern pandas has used, for instance.)
   - First deprecate and warn that `overwrite`, `set_default`, `quiet` will not be accepted as positional arguments.
   - Then in a later version remove the deprecated functionality.

Can we at least do (1) here?

Reply:
Also, the added argument and CLI option are changes in the user API, so they should be mentioned in the release notes.